blockchain/genesis_alloc.go

```diff
@@ -31,29 +31,6 @@ const testnetAllocData = "\xda\u0654\x19t\x18\x1a?\x1bE+\x00\xec\u06ab\x19\xa2j\
 
 var (
 	baobabGenesisJson = []byte(`{
```

Comment:
After this change, `baobabGenesisJson` doesn't contain a full genesis. Currently, the value is used in a safe way. But, I am afraid that someone uses this value confusing with full genesis.

Reply:
You're right.. I restored genesis_alloc.go chainconfig part.